docs/README.md

```diff
@@ -19,6 +20,24 @@ Quotes from the community:
 
 > reusable, well abstracted solution to run LLMs - [Mike Ensor](https://www.linkedin.com/posts/mikeensor_gcp-solutions-public-retail-edge-available-cluster-traits-activity-7237515920259104769-vBs9?utm_source=share&utm_medium=member_desktop)
 
+## Why KubeAI?
+
+### Better performance at scale
```

Comment:
+Intro: `When running multiple replicas of a serving engine such as vLLM, performance under production traffic is heavily influence by the load balancing strategy.`

Reply:
I will leave this as is but please do a follow up PR to modify the README.